esx_service/cli/vmdkops_admin.py

```diff
@@ -248,13 +253,14 @@ def commands():
                                 }
                             }
                         },
+
                         'ls': {
                             'help': "list VMs in a tenant",
                             'func': tenant_vm_ls,
                             'args': {
                                 '--name': {
                                     'help': "Tenant to list the VMs for",
-                                    'required': False
+                                    'required': True
```

Comment:
information on VMs and tenants is different. If you have some specific format in mind, please give example on how it would look like - FWIW I can't visualize your suggestion